libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/TabsView.tsx

```diff
@@ -120,6 +121,20 @@ export class TabsView extends React.PureComponent<
               className={classNames.section}
               styles={{ root: { boxShadow: DefaultEffects.elevation4 } }}
             >
+              {t.key === GlobalTabKeys.VisionTab &&
+                this.props.visionModelExplanationData && (
+                  <>
+                    <div className={classNames.sectionHeader}>
```

Comment:
try to use Stack instead of div, like here

Reply:
I was just copying all of the other components which use <div> and <h3>. Should I still use Stack even if all the other components aren't, or should I update them all to be Stacks?

Reply:
Yes please use Stack wherever possible.  The other components/examples are probably wrong if they use div.  Stack is essentially div but with fluentui sprinkled in to fit our accessibility/style requirements.  So please try to use Stack wherever possible instead of div.